storage_drivers/azure/api/azure.go

```diff
@@ -129,17 +131,49 @@ type Client struct {
 }
 
 // NewDriver is a factory method for creating a new SDK interface.
-func NewDriver(config ClientConfig) (Azure, error) {
+func NewDriver(ctx context.Context, config ClientConfig) (Azure, error) {
```

Comment:
the logic should be if backend secret is defined, then only use backend secret to get credentials, if not, then use `/etc/kubernetes/azure.json` by default.

Reply:
OK, currect logic is just the opposite.